### PR TITLE
Page Order and Language

### PR DESCRIPTION
**Pull request by Diab98**
The order of this page is very confusing. There are many repeating titles such as "Filter by Boolean expression" and so on that provide more information than the previous sections with the same title. This makes the page very long and dense.  I propose providing an overview/ table of contents for the tutorial at the beginning that depict a logical and cohesive progression.  For example: 
1. A brief description of what Kusto Query language is. Following that, a seamless continuation into the example used ('StormEvents').
2. Counting.
3. Different operators for picking columns: project/ take. 
4. etc.  In addition, I would rephrase and organize various sections. The language could be more user friendly in terms of syntax/grammar/ visual presentation.

### CODE REVIEW

**Comment by shsagir** [comment]
Learn Build status updates of commit _[393daaa](https://github.com/Diab98/dataexplorer-docs/commits/393daaad2a68d73b671a8501dc08f20cc631a5e8)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/kusto/query/tutorial.md](https://github.com/Diab98/dataexplorer-docs/blob/patch-1/data-explorer/kusto/query/tutorial.md) | :warning:Warning | | [Details](#user-content-55ea57450a69e2f75be0438b4e7d04cb-88e69cb6)

<a id="55ea57450a69e2f75be0438b4e7d04cb-88e69cb6"></a>
### [data-explorer/kusto/query/tutorial.md](https://github.com/Diab98/dataexplorer-docs/blob/patch-1/data-explorer/kusto/query/tutorial.md)
 - **Line 0, Column 0**: **[Warning: pivot-id-unused - [See documentation](https://review.learn.microsoft.com/help/contribute/validation-ref/pivot-id-unused?branch=main)]** `````Pivot ID 'azuredataexplorer' is defined in 'kql-flavors' group in 'zone-pivot-groups.yml', but not used. You must use all pivot IDs in a pivot group to avoid blank tabs.`````
 - **Line 21, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Count rows''. Each second level heading (H2) in an article must be unique.`````
 - **Line 39, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Select a subset of columns: project''. Each second level heading (H2) in an article must be unique.`````
 - **Line 44, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Show n rows: take''. Each second level heading (H2) in an article must be unique.`````
 - **Line 68, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Filter by Boolean expression: where''. Each second level heading (H2) in an article must be unique.`````
 - **Line 86, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Order results: sort, top''. Each second level heading (H2) in an article must be unique.`````
 - **Line 120, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Compute derived columns: extend''. Each second level heading (H2) in an article must be unique.`````
 - **Line 161, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Aggregate groups of rows: summarize''. Each second level heading (H2) in an article must be unique.`````
 - **Line 198, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Summarize by scalar values''. Each second level heading (H2) in an article must be unique.`````
 - **Line 225, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Display a chart or table: render''. Each second level heading (H2) in an article must be unique.`````
 - **Line 456, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Assign a result to a variable: let''. Each second level heading (H2) in an article must be unique.`````
 - **Line 514, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Next steps''. Each second level heading (H2) in an article must be unique.`````
 - **Line 528, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Count rows''. Each second level heading (H2) in an article must be unique.`````
 - **Line 545, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Filter by Boolean expression: where''. Each second level heading (H2) in an article must be unique.`````
 - **Line 562, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Select a subset of columns: project''. Each second level heading (H2) in an article must be unique.`````
 - **Line 575, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Show n rows: take''. Each second level heading (H2) in an article must be unique.`````
 - **Line 587, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Order results: sort, top''. Each second level heading (H2) in an article must be unique.`````
 - **Line 608, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Compute derived columns: extend''. Each second level heading (H2) in an article must be unique.`````
 - **Line 623, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Aggregate groups of rows: summarize''. Each second level heading (H2) in an article must be unique.`````
 - **Line 636, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Summarize by scalar values''. Each second level heading (H2) in an article must be unique.`````
 - **Line 651, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Display a chart or table: render''. Each second level heading (H2) in an article must be unique.`````
 - **Line 702, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Assign a result to a variable: let''. Each second level heading (H2) in an article must be unique.`````
 - **Line 719, Column 1**: **[Warning: duplicate-h2s - [See documentation](https://review.learn.microsoft.com/en-us/help/contribute/validation-ref/duplicate-h2s?branch=master)]** `````Duplicate heading: 'H2 'Next steps''. Each second level heading (H2) in an article must be unique.`````

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202209281021149602-1418/BuildReport?accessString=1825a880720f823e5179f0e2c0dce79006805ea916a293ab907a3fba09a4f9e0).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.learn.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Learn Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings or suggestions unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.learn.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by jborsecnik** [comment]
@shsagir, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by YaelSchuster** [comment]
#please-close
